tests/data/instances/mc_tuples_with_background_instance_test.py

```diff
@@ -0,0 +1,284 @@
+# pylint: disable=no-self-use,invalid-name
+
+from unittest import TestCase
+import numpy as np
+
+from deep_qa.data.data_indexer import DataIndexer
+from deep_qa.data.dataset import TextDataset
+from deep_qa.data.instances.mc_tuples_with_background_instance import MCTuplesWithBackgroundInstance, \
+    IndexedMCTuplesWithBackgroundInstance
```

Comment:
I agree.  Though, if you rename this to `TupleInferenceInstance` (including moving the file), the line might be short enough to put these on the same line.